lib/SILGen/SILGenProlog.cpp

```diff
@@ -284,20 +284,6 @@ struct ArgumentInitHelper {
         gen.B.createDebugValueAddr(loc, argrv.getValue(), {vd->isLet(), ArgNo});
       else
         gen.B.createDebugValue(loc, argrv.getValue(), {vd->isLet(), ArgNo});
-    } else {
-      // If the variable is mutable, we need to copy or move the argument
```

Comment:
Can you add an assert that vd->isLet() instead of just falling through the if/else if?